src/main/java/net/sf/jsqlparser/statement/select/PlainSelect.java

```diff
@@ -465,7 +469,15 @@ public void setMySqlSqlCalcFoundRows(boolean mySqlCalcFoundRows) {
         this.mySqlSqlCalcFoundRows = mySqlCalcFoundRows;
     }
 
+    public void setMySqlSqlNoCache(boolean sqlNoCacheFlagSet) {
```

Comment:
Should the field name also be mySql like the other MySQL specific one? Or do other dbs provide a no cache mode

Reply:
Thanks for the review.
As far as I know, there are no other databases which support this flag. Seems like an extension of MySQL.

Reply:
Ah, then id name the field like the other MySQL one (makes it clearer in this class)

Reply:
I'm not sure I understood the suggestion. can you clarify please?

Reply:
Yeah, since this field is for mySQL support only, we should rename it to be `mySqlSqlNoCacheFlag` like the `mySqlSqlCalcRows` field (that way we can distinguish that this is MySQL specific.